src/main/java/io/nats/client/api/AccountLimits.java

```diff
@@ -105,18 +107,4 @@ public long getStorageMaxStreamBytes() {
     public boolean isMaxBytesRequired() {
         return maxBytesRequired;
     }
-
-    @Override
-    public String toString() {
-        return "AccountLimits{" +
-            "maxMemory=" + maxMemory +
-            ", maxStorage=" + maxStorage +
-            ", maxStreams=" + maxStreams +
-            ", maxConsumers=" + maxConsumers +
-            ", maxAckPending=" + maxAckPending +
-            ", memoryMaxStreamBytes=" + memoryMaxStreamBytes +
-            ", storageMaxStreamBytes=" + storageMaxStreamBytes +
-            ", maxBytesRequired=" + maxBytesRequired +
-            '}';
-    }
```

Comment:
toString is no longer required in many places because there is automatic toString in base classes and in the JsonValue